ecosystem/indexer-grpc/indexer-grpc-fullnode/src/stream_coordinator.rs

```diff
@@ -187,9 +187,21 @@ impl IndexerStreamCoordinator {
         // Stage 3: send responses to stream
         let sending_start_time = std::time::Instant::now();
         for response in responses {
-            if self.transactions_sender.send(Ok(response)).await.is_err() {
-                // Error from closed channel. This means the client has disconnected.
-                return vec![];
+            match self.transactions_sender.send(Ok(response)) {
+                Ok(_) => (),
+                Err(e) => {
+                    // Error from closed channel. This means the client has disconnected.
+                    error!(
+                        start_version = first_version,
+                        end_version = end_version,
+                        num_transaction,
+                        highest_known_version,
+                        duration_in_secs = sending_start_time.elapsed().as_secs_f64(),
+                        error = format!("{:?}", e),
```

Comment:
if potential error msgs could be verbose, we might wanna consider trim it down a bit , could set a pretty conservative number to begin with a

Reply:
i don't think we trim the error in other places in fullnode/grpc so it should be fine to leave as is. better to have the full error message than truncated version haha